app/views/pages/dsi-account-request.html.slim

```diff
@@ -18,6 +18,7 @@
     ul.govuk-list.govuk-list--bullet class="govuk-!-margin-bottom-8"
       li at a state-funded school or trust in England, or
       li if you work at a local authority and list job vacancies on behalf of schools
+    p.govuk-body #{govuk_link_to("Check you're eligible to list roles on Teaching Vacancies", "https://teaching-vacancies.service.gov.uk/get-help-hiring/how-to-setup-your-account/check-your-eligibility-to-advertise-roles")}.
```

Comment:
Unsure why when I click on this link, it says 'page not found'? There's also a big gap between this link and the above paragraph?